lib/exchange_clients/bitfinex/recv/data.js

```diff
@@ -20,7 +22,7 @@ module.exports = (exa, msg) => {
   // give subscription promises a chance to resolve
   setTimeout(() => {
     for (let i = 0; i < dataListeners.length; i += 1) {
-      dataListeners[i](chanID, msg)
+      dataListeners[i]([msgType, transform(msgType, msgData)])
     }
   }, 0)
```

Comment:
maybe we can use `setImmediate` or `nextTick` instead?

Reply:
I'm not sure if this won't break anything since `setImmediate`  and `nextTick` works differently